.github/workflows/integration-test.yml

```diff
@@ -21,6 +25,8 @@ defaults:
 jobs:
   run-integration-test:
     name: Run Integration Tests
+    # do not run the tests with manifests installation and scheduling gates
```

Comment:
why do scheduling gates and manifest cannot go together?

Reply:
They surely can, but as we plan to remove the support of installation via manifests and therefore also the integration tests and all the pipelines, it does not make sense to invest effort (not much but still..) to testing the exact same thing with manifests and helm installation. It's enough to use helm install to test that the feature works correctly.